### ISSUE TITLE
server crash onChannelMessage at lib/fedserver.js

### ISSUE THREAD

**Issue by tlenex** [capriza/node-busmq#36]
By some miracle there was a `JSON.parse()` error when my app received message `undefined`. Of course my server crashed. Perhaps simple `try-catch` block will prevent this.
```
1113867 undefined:1
1113868 undefined
1113869 ^
1113870 
1113871 SyntaxError: Unexpected token u
1113872     at Object.parse (native)
1113873     at WSMuxChannel.onChannelMessage (/home/server/messagebus/dist/node_modules/busmq/lib/fedserver.js:137:16)
1113874     at WSMuxChannel.g (events.js:260:16)
1113875     at emitOne (events.js:77:13)
1113876     at WSMuxChannel.emit (events.js:169:7)
1113877     at WSMux._onMessage (/home/server/messagebus/dist/node_modules/busmq/lib/wsmux.js:335:11)
1113878     at WebSocket.onMessage (/home/server/messagebus/dist/node_modules/busmq/lib/wsmux.js:148:11)
1113879     at emitTwo (events.js:87:13)
1113880     at WebSocket.emit (events.js:172:7)
1113881     at Receiver.ontext (/home/server/messagebus/dist/node_modules/ws/lib/WebSocket.js:816:10)
```

**Comment by fujifish** [comment]
Interesting. Any way you can reproduce this?
In any case, I'll add a try-catch and emit 'error' message if it happens

**Comment by fujifish** [comment]
published version 1.5.3 with the fix

**Comment by tlenex** [comment]
Sorry, I don't know how that happend. If I dig something out I'll let you know
